lib/processor.dart

```diff
@@ -16,16 +16,49 @@ class Processor {
     this.multiTokenTransformerFactories = const [],
   });
 
-  void process() {
+  void process([List<String> filteredSets = const []]) {
```

Comment:
Can we make this a named parameter to make it easier to add more parameters later?

Unless the library is opposed to named parameters